### DISCUSSION TITLE
Performace comparision

### DISCUSSION THREAD

**Discussion by tureckova** [obss/sahi#338] [General]
Hello,
Once again, thank you for your great job there.
I wonder whether you made some performance comparison of the results with different predict parameters set up. And also the comparison of results with and without your library. It would be good to have a reference on setting up the parameters, at least at the beginning.
Thank you for your reaction :smiley_cat:

**Comment by fcakyon** [comment]
@tureckova it is a work in progress, i am planning to publish the results with my professor as a paper in a month :rocket:

**Reply by tureckova** [reply]
OK, cool. I'll wait a bit then. I wish you good luck with reviewers. :crossed_fingers:

**Comment by tureckova** [comment]
Hello, please, is there some reference for the NMM and GREEDYNMM postprocessing algorithms? Or is it something new you will describe in your paper? I tried and failed to find it both on the web and in research papers.
Thank you very much for your kind direction.

**Reply by fcakyon** [reply]
These methods are newly introduced in `sahi`. They are better used with high confidence predictions (>0.4 confidence).

In literature, confidence=0.01 is used to increase the COCO AP but in practice higher confidence thresholds are used. So NMM and GREEDYNMM methods perform great in practical usage but in terms of COCO AP, NMS would yield better results.
